ChiTech/ChiPhysics/FieldFunction/lua/physics_exportfieldfunc.cc

```diff
@@ -126,13 +126,16 @@ int chiExportMultiFieldFunctionToVTKG(lua_State *L)
   catch(const std::out_of_range& o)
   {
     chi_log.Log(LOG_ALLERROR)
-      << "Invalid field function handle in chiPhysicsExportFieldFunctionToVTK";
+      << "Invalid field function handle in chiExportMultiFieldFunctionToVTKG";
     exit(EXIT_FAILURE);
   }
 
 //  ff->ExportToVTKG(base_name,field_name);
 
 //  ff->ExportMultiToVTKG(ff_slave,base_name,field_name);
+
+  throw std::logic_error("chiExportMultiFieldFunctionToVTKG : not implemented.");
+
```

Comment:
The choice on how to proceed in this case may be related to the upcoming refactor of the field functions mentioned in pull request #152. However, if it is known at this time that there is no intention to implement the underlying C++ functionality upon which the Lua wrapper is based, then I propose to remove the Lua wrapper as part of this pull request. In the meantime, I have added the exception as a precaution.

Reply:
This particular lua function needs to be removed. I don't know what it is doing there. Might have lingered there during development. Can you please just remove it?

Reply:
Removed.